docs/narrative_data_model.md

```diff
@@ -41,13 +41,15 @@ I will log in and be "onboarded" via the UI; I will upload the case records of t
 > | Amaury | Mòsi | 2012-09-11 | Little Leaf Day School | 60101 | DuPage | Bronze | 4567890 | 11 | 7 | 2020-02-04 | 2021-02-04 | $12 | Weekly |  
 > 
 > The remaining childrren will each have their own record with unique case numbers
-  
+    
 **TO BE IMPLEMENTED**: At the end of onboarding, I will have entered data to create 4 **Approvals** (one for each family), 6 **Children**, 6 **ChildApprovals**, 2 **Businesses**
 
 The backend will then do the following:  
 
 - based on **Child**'s age, **County** where care is received, QRIS rating of the **Business** [some other info?] - associate a **SubsidyRule** with each **ChildApproval**
 
+- check on **Child**'s address because the provider will not receive a subsidy for a child that is not a resident of Illinois 
```

Comment:
```suggestion
- check on **Child**'s address because the provider will not receive a subsidy for a child that is not a resident of Illinois [TO BE IMPLEMENTED]
```